### PR TITLE
TASK-1562 Pedigree graph with a three-generation family fails to render properly in Family browser detailed tab

### PR DESCRIPTION
**Pull request by jtarraga**
Implement pedigree graph.

### CODE REVIEW

**Comment by jtarraga** [comment]
Task linked: [TASK-1562 Pedigree graph with a three-generation family fails to render properly in Family browser detailed tab](https://app.clickup.com/t/36631768/TASK-1562)

File: opencga-core/src/main/java/org/opencb/opencga/core/models/family/Family.java [hunk 1]
```diff
@@ -153,6 +153,9 @@ public class Family extends Annotable {
             description = FieldConstants.FAMILY_ROLES)
     private Map<String, Map<String, FamiliarRelationship>> roles;
 
+    @DataField(id = "b64PedigreeGraph", description = FieldConstants.FAMILY_PEDIGREE_GRAPH)
```
Comment (pfurio) [inline opencga-core/src/main/java/org/opencb/opencga/core/models/family/Family.java:156]: Is it really going to be called that way?

**Review by pfurio — APPROVED** [review]
I'm not sure which field name I would use to contain the base 64 pedigree graph image, but I would certainly not call it _b64PedigreeGraph_. If everyone's happy with that, we can go ahead.

**Review by pfurio — APPROVED** [review]

File: opencga-analysis/src/test/java/org/opencb/opencga/analysis/family/FamilyAnalysisTest.java [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+import org.opencb.opencga.storage.core.StorageEngineFactory;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class FamilyAnalysisTest extends GenericTest {
```
Comment (j-coll) [inline opencga-analysis/src/test/java/org/opencb/opencga/analysis/family/FamilyAnalysisTest.java:39]: Reminder : We'll need to add the `@Category` to this test once the categories are merged

**Review by j-coll — APPROVED** [review]

**Pull request merged**